qnapstats/qnap_stats.py

```diff
@@ -112,7 +120,7 @@ def _handle_response(self, resp, force_list=None):
         self._debuglog("Response Text: " + resp.text)
         data = xmltodict.parse(resp.content, force_list=force_list)['QDocRoot']
 
-        auth_passed = data['authPassed']
+        auth_passed = data.get('authPassed')
```

Comment:
I don't know enough about Python to determine is this is correct or not, but the tests seem to pass so I'll trust you on this one :)

Reply:
It's to avoid ValueError if key `authPassed`  not exists in data. data.get(`authPassed`) will return None if not exists in data.